### PR TITLE
remove release tools, update gitignore

### PR DESCRIPTION
**Pull request by krishchow**
This PR removes the release-tools folder and switches to using the release tools from the COSI spec repository.

### CODE REVIEW

**Comment by krishchow** [comment]
cc @brahmaroutu @rrati

**Comment by brahmaroutu** [comment]
Please add the Makefile

File: .prow.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+. release-tools/prow.sh
+
+main
```
Comment (wlan0) [inline .prow.sh:5]: pls add a newline at the end

File: Dockerfile [hunk 2]
```diff
@@ -13,11 +11,7 @@
...
-    echo "Repo uses 'dep' for vendoring."
-    (set -x; dep ensure)
-elif [ -f go.mod ]; then
-    release-tools/verify-go-version.sh "go"
-    (set -x; env GO111MODULE=on go mod tidy && env GO111MODULE=on go mod vendor)
-fi
+FROM gcr.io/distroless/static:latest
+WORKDIR /
+COPY ./bin/csi-adapter csi-adapter
+ENTRYPOINT ["/csi-adapter"]
```
Comment (wlan0) [inline Dockerfile:17]: pls add a new line at the end

**Comment by brahmaroutu** [comment]
/lgtm
Please address the comments in the other pr, once you rebase this.

**Pull request merged**